nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/record/RecordFieldType.java

```diff
@@ -72,6 +72,11 @@
      */
     DOUBLE("double", FLOAT),
 
+    /**
+     * A big decimal field type. Fields of this type use a {@code java.math.BigDecimal} value.
+     */
+    BIGDECIMAL("bigdecimal", FLOAT, DOUBLE),
```

Comment:
I was unluckily looking for BigInt enum usage instead of usage of the whole RecordFieldType. I added them, thank you for finding this. Also, it's a question for me if BigInt should be explicitly handled in these cases.